### PR TITLE
Fix zero nonce scenario for MultiESDTNFTTransfer

### PR DESCRIPTION
**Pull request by popenta**

### CODE REVIEW

**Review by andreibancioiu — APPROVED** [review]

File: transaction_decoder/transaction_decoder.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def get_multi_transfer_metadata(
 
                 tx_metadata.value = value
                 tx_metadata.properties = TokenTransferProperties()
-                tx_metadata.properties.collection = identifier
+                tx_metadata.properties.token = identifier
```
[the remark this answers is not in the document]
Comment (popenta) [reply transaction_decoder/transaction_decoder.py:197]: Bumped the minor version.

File: transaction_decoder/transaction_decoder.py [hunk 2]
```diff
@@ -180,7 +180,7 @@ def get_multi_transfer_metadata(
             value = self.hex_to_big_int(args[index])
             index += 1
 
-            if nonce:
+            if nonce and self.hex_to_number(nonce) > 0:
```
[the remark this answers is not in the document]
Comment (popenta) [reply transaction_decoder/transaction_decoder.py:183]: Done!

Comment (andreibancioiu) [inline transaction_decoder/transaction_decoder.py:183]: `if nonce` not needed anymore theoretically.

**Review by andreibancioiu — APPROVED** [review]

File: transaction_decoder/transaction_decoder.py [hunk 3]
```diff
@@ -279,9 +279,10 @@ def base64_decode(self, s: str) -> str:
         return base64.b64decode(s.encode("utf-8")).decode("utf-8")
 
     def hex_to_number(self, hex: str) -> int:
-        return int(hex, 16)
+        return int(hex or "00", 16)
 
     def bech32_encode(self, address: str) -> str:
         pub_key = bytes.fromhex(address)
         words = convertbits(pub_key, 8, 5)
+        assert words is not None
```
Comment (ssd04) [inline transaction_decoder/transaction_decoder.py:287]: not sure about best practices when using assert, was this just for testing/debugging? or we usually use assert in production code

**Review by ssd04 — APPROVED** [review]

File: transaction_decoder/transaction_decoder.py (hunk 3 above)
Comment (popenta) [reply transaction_decoder/transaction_decoder.py:287]: It was not for testing/debugging, it's there to fix the linter error.

**Pull request merged**